src/containers/VisitPage.js

```diff
@@ -355,20 +357,18 @@ class VisitPage extends Component {
           <FeedbackAnytime openFeedback={ this.openFeedback } />
         </Container>
 
-        { 
-          distrustConfirmed === false ? (
-            <PredictionsWarning
-              distrustConfirmed       = { distrustConfirmed }
-              toggleDistrustConfirmed = { this.props.funcs.toggleDistrustConfirmed }
-              translations={{ ...translations.warningModal }} />
-          ) : (
-            null
-          )
-        }
+        { (distrustConfirmed === false) ? (
```

Comment:
(same question as above)

Reply:
Same answer. This isn't a super complex one, though it's a bit more complex and the parens help even here. I also think it makes it look even more like a regular if statement, which is what I find easiest to read.

Reply:
@turnerhayes : Would we remove the parens here, or do they stay because this has an operation?

Reply:
Debatable--I would say equality is a simple enough operation, but that does get into the question of "what's simple enough" and that's probably not something we want to wade into.

Would this be okay to replace with `!distrustConfirmed`? Or can it have falsy-but-not-`false` values?

Reply:
I can switch it up

Reply:
I don't think it has falsy/truthy values...